packages/vector_graphics_compiler/lib/src/svg/parser.dart

```diff
@@ -980,7 +980,6 @@ class SvgParser {
   }) {
     return numbers.parseDoubleWithUnits(
       rawDouble,
-      tryParse: tryParse,
```

Comment:
This has recreated the same problem, just one level up; the `tryParse` parameter is now ignored in this method.

If removing `tryParse` is in fact the correct behavior, it needs to be done in a comprehensive and consistent way. Any version that leaves options at other levels of the code that are silently ignored isn't going to pass review.